image/image.go

```diff
@@ -119,13 +109,18 @@ func (i *genericImage) DockerTar() ([]byte, error) {
 // will support v1 one day...
 type genericManifest interface {
 	String() string
-	GetLayers() []string
+	Layers() []string
+	Config() ([]byte, error)
+	ImageInspectInfo() (*types.ImageInspectInfo, error)
```

Comment:
With `ImageInspectInfo` separate per schema, it seems unnecessary to have `Config` as a separate method, at least for now. OTOH it seems plausible enough that it will be needed in the future…